src/components/App.jsx

```diff
@@ -24,41 +25,51 @@ class App extends Component {
     this.props.fetchUserAccount(userAccountApiService, username);
   }
 
+  renderContent() {
+    if (!this.props.loaded) {
```

Comment:
Pulling this out into a separate function and conditionally returning on `loaded` allows us to defer calling `fetchProfile` in ConnectedProfilePage until we've gotten the userAccount data.  This is the serialization I was talking about in the PR description.  We don't even want to show the ConnectedProfilePage if the user isn't authenticated, so I didn't want to bother making requests for the profile data here in componentDidMount() before we know if we even need to.